src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+    protected String task;
+    protected boolean isDone;
+
+    public Task() {
+        setTask("");
+        setDone(false);
+    }
+
+    public Task(String task) {
+        setTask(task);
+        setDone(false);
+    }
+
+    public String getTask() {
+        return task;
+    }
+
+    public boolean isDone() {
```

Comment:
You could consider giving verb-like names for the method.
```suggestion
    public boolean checkIsDone() {
```